app/controllers/api/v0/api_controller.rb

```diff
@@ -54,6 +63,7 @@ def authenticate_with_api_key
     # see <https://www.slideshare.net/NickMalcolm/timing-attacks-and-ruby-on-rails>
     if ActiveSupport::SecurityUtils.secure_compare(api_secret.secret, api_key) # rubocop:disable Style/GuardClause
       @user = api_secret.user
+      @user
```

Comment:
Won't it return `@user` anyway without this line?

Reply:
Ahah yes, I think I've made it too explicit 😂